server/conf/cms/secrets.sample.py

```diff
@@ -1,351 +1,50 @@
-# TACC WMA SAD CMS SITE: CEP.TACC.UTEXAS.EDU
```

Comment:
for the file name change (secrets.sample.py -> settings_custom.sample.py), the README.md instructions need to be updated

Reply:
Hrm. README **does** need update, but the file was not renamed; `settings_custom.py` is new and has a purpose[^1], but I think a `sample` for it need not exist here. I'll make a commit. If it doesn't work, I'll rethink.

Reply:
Does 7baaf3c clarify CMS's `settings_custom`?

Reply:
👍 


yeah, sorry.  wrote the comment too fast and then updated it.

Reply:
Cool, update made. That file is deleted too, so tell me what you think.